dunstrc

```diff
@@ -132,6 +132,10 @@
     # Possible values are "left", "center" and "right".
     alignment = left
 
+    # Vertical alignment of message text and icon.
+    # Possible values are "top", "center" and "bottom".
+    content_alignment = center
```

Comment:
Lets call it `vertical_alignment`, as IMO, `content_alignment` is a bit confusing when there's already a setting called `alignment`.

Reply:
I thought about it and came to the conclusion, that one could use `vertical_alignment` for alignment of the complete message window. And while `alignment` describes the alignment of the text, `content_alignment` then describes alignment of the message's content. But I concur, that it is not really obvious.

If you prefer, I will change it. I am not insisting on `content_alignment` in any way. :)

Reply:
> I thought about it and came to the conclusion, that one could use vertical_alignment for alignment of the complete message window

In #336 we've decided that we'll call the new setting defining the actual window position `origin`, so there's no issue with using 'vertical_alignment' here.